From 1fbd0bdab6688cb7882f5934e6220ce9b36fe3e9 Mon Sep 17 00:00:00 2001
From: Emilian Bold <emi@apache.org>
Date: Thu, 8 Nov 2018 20:36:56 +0200
Subject: [PATCH 12/32] Using old version of modules, not the newer ones after
 the cherry-picked retina commits
License: Copyright (c) 2018, 2019 Emilian Marius Bold
 
 Permission to use, copy, modify, and distribute this software for any
 purpose with or without fee is hereby granted, provided that the above
 copyright notice and this permission notice appear in all copies.
 
 THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
 WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
 MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
 ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.

---
 platform/o.n.swing.tabcontrol/nbproject/project.xml | 2 +-
 platform/openide.awt/nbproject/project.xml          | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/platform/o.n.swing.tabcontrol/nbproject/project.xml b/platform/o.n.swing.tabcontrol/nbproject/project.xml
index 5c88d9844f..7412416029 100644
--- a/platform/o.n.swing.tabcontrol/nbproject/project.xml
+++ b/platform/o.n.swing.tabcontrol/nbproject/project.xml
@@ -38,7 +38,7 @@
                     <build-prerequisite/>
                     <compile-dependency/>
                     <run-dependency>
-                        <specification-version>9.12</specification-version>
+                        <specification-version>9.3</specification-version>
                     </run-dependency>
                 </dependency>
                 <dependency>
diff --git a/platform/openide.awt/nbproject/project.xml b/platform/openide.awt/nbproject/project.xml
index 0398c101d6..7d008a9f94 100644
--- a/platform/openide.awt/nbproject/project.xml
+++ b/platform/openide.awt/nbproject/project.xml
@@ -47,7 +47,7 @@
                     <build-prerequisite/>
                     <compile-dependency/>
                     <run-dependency>
-                        <specification-version>9.12</specification-version>
+                        <specification-version>9.11</specification-version>
                     </run-dependency>
                 </dependency>
                 <dependency>
-- 
2.21.0

